src/src/AdminBundle/Resources/views/base.html.twig

```diff
@@ -4,51 +4,25 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Nobelists</title>
-    <!-- Bootstrap core CSS -->
-    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" rel="stylesheet">
-    <!-- Custom styles for this template -->
 
-    {# todo: move to assets! #}
-    <style>
-        body {
-            padding-top: 5rem;
-        }
-        .starter-template {
-            padding: 3rem 1.5rem;
-            text-align: center;
-        }
-        form label {
-            text-align: right;
-        }
-        .tab-pane {
-            margin-top: 20px;
-        }
-    </style>
+    {% stylesheets 'assets/css/*' filter='cssrewrite' %}
+        <link rel="stylesheet" href="{{ asset_url }}" />
+    {% endstylesheets %}
 </head>
 <body>
     {% block navigation %}
     {% endblock %}
     <main role="main" class="container">
         {% block content %}
         {% endblock %}
-        {% block modals %}
-        {% endblock %}
     </main>
 
-    <!-- Bootstrap core JavaScript
-    ================================================== -->
-    <!-- Placed at the end of the document so the pages load faster -->
-    <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js"
-            integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n"
-            crossorigin="anonymous">
-    </script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js"
-            integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb"
-            crossorigin="anonymous">
-    </script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js"
-            integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn"
-            crossorigin="anonymous">
-    </script>
+    {% javascripts
+        'assets/js/jquery-3.1.1.slim.min.js'
+        'assets/js/tether.min.js'
+        'assets/js/*'
```

Comment:
Preco sa vymenuvavaju jquery a tether zvlast?
Wildcard by ich mal zachytit, nie?

Reply:
wildecast sice jquery a thether zachyti ale poradie sa urci abecedne. To je vsak problem, kedze bootstrap ocakava uz funkcne jQuery. Explicitne vymenovanie niektorych suborov garantuje ich poradie.